### PR TITLE
Fix naming inconsistency in Instance claim map

### LINKED ISSUE #43: Possible naming inconsistency in Instance claim map
Currently, `instance-claim-map` has:
https://github.com/ietf-rats/ietf-corim-cddl/blob/3be51928b05e6a8034aef01b80645d936729ba4c/concise-mid-tag.cddl#L76
Should that be `comid.element-name` instead?

### PR DESCRIPTION
**Pull request by thomas-fossati**
Fixes #43

### CODE REVIEW

File: comid-code-points.cddl [hunk 1]
```diff
@@ -75,7 +75,7 @@ comid.patches = 5
 comid.version-scheme = 1
 
 ; endorsed-claim-map and reference-claim-map
-; (also identity-claim-map)
+; (also identity-claim-map and instance-value-map)
```
Comment (yogeshbdeshpande) [inline comid-code-points.cddl:78]: The correct name in actual CDDL is instance-claim-map, isn't it ?

Comment (thomas-fossati) [inline comid-code-points.cddl:78]:
```suggestion
; (also identity-claim-map and instance-claim-map)
```

File: comid-code-points.cddl [hunk 2]
```diff
@@ -90,8 +90,7 @@ comid.instance-claims = 3
 comid.device-id = 1
 comid.key-material = 2
 
-; instance-value-map and $instance-value-choice
-comid.instance-element-name = 0
+; $instance-value-choice
```
Comment (yogeshbdeshpande) [inline comid-code-points.cddl:93]: There is no such thing as $instance-value-choice, can we say ; member of $$instance-value-group-choice ??

Comment (thomas-fossati) [inline comid-code-points.cddl:93]:
```suggestion
; $$instance-value-group-choice
```

**Review by yogeshbdeshpande — APPROVED** [review]

**Review by nedmsmith — COMMENTED** [review]
+1

**Pull request merged**